start.js

```diff
@@ -4,12 +4,14 @@ const { dependencies } = require('./package');
 const bodyParser = require('body-parser');
 const apihead = require('./headers/API');
 
+const PORT = process.env.PORT || 90;

```

Comment:
I was going to make a config json file for this type of stuff so this won't be needed

Reply:
Eh, environment variables are useful too. I'd still go with something like this.